pkg/ui/workspaces/cluster-ui/src/graphs/utils/domain.ts

```diff
@@ -320,6 +324,9 @@ export function calculateYAxisDomain(
       return ComputeDurationAxisDomain(yExtent);
     case AxisUnits.Percentage:
       return ComputePercentageAxisDomain(yExtent[0], yExtent[1]);
+    case AxisUnits.DurationMillis:
+      return ComputeDurationAxisDomain(yExtent);
+      //return ComputeDurationAxisDomain([min(allDatapoints),max([max(allDatapoints), 1000000])]);
```

Comment:
Should this be commented out?

Reply:
ah thankyou, I was doing back and forth for screenshots 😬 . that chunk should be restored